### DISCUSSION TITLE
Storage Template for video's

### DISCUSSION THREAD

**Discussion by mohandarsi** [immich-app/immich#14525] [Q&A]
I enabled stroage template in my immich server, but after uploading photos and videos,
 But i see videos are not following storage tample format.
videos are getting stored under enocded_video folder with default folder names.
How can i enable videos to follow storage template pattern??

![Screenshot from 2024-12-06 16-46-43](https://github.com/user-attachments/assets/2a68e89e-f4c0-421a-b4ba-8fccd9008896)

**Comment by bo0tzz** [comment]
Storage template applies to videos as well. All the uploaded files go in either `upload` or `library`. `encoded-video` is a separate folder with files generated by Immich, that you don't need to worry about.

**Reply by mohandarsi** [reply]
Do you mean, these are duplicates created by Immich and the copy of original exists in the `library` or `upload` folder?? 
if so, can i delete this folder safely ??

**Reply by bo0tzz** [reply]
They aren't duplicates, and you can't just delete them from the folder. https://immich.app/docs/FAQ#why-does-immich-transcode-my-videos-to-a-lower-quality
